tapeagents/orchestrator.py

```diff
@@ -93,6 +95,8 @@ def main_loop(
     """
 
     def _implementation():
+        if is_debug_mode():
```

Comment:
not sure we want this here

Reply:
We do, otherwise we forced to update the code of the orchestrator every time we want to observe steps in a log